cloudfoundry-client-reactor/src/main/java/org/cloudfoundry/reactor/tokenprovider/_PasswordGrantTokenProvider.java

```diff
@@ -36,6 +36,11 @@
      * The username
      */
     abstract String getUsername();
+    
+    /**
+     * The login_hint
```

Comment:
Javadoc should use English, e.g. "The identity provider to be used", or at least "The login hint"